modules/fastcv/test/test_moments.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+*/
+
+#include "opencv2/ts.hpp"
+#include "opencv2/moments.hpp"
+
+namespace opencv_test { namespace {
+
+typedef testing::TestWithParam<tuple<bool,Size,int>> fcv_momentsTest;
+
+TEST_P(fcv_momentsTest, accuracy)
+{
+    const bool binaryImage = get<0>(GetParam());
+    const Size srcSize = get<1>(GetParam());
+    const MatDepth srcType = get<2>(GetParam());
+    Mat src(srcSize, srcType);
+
+	for(int j = 0; j < srcSize.width; ++j)
```

Comment:
we have `cv::RNG::fill()` for that

Reply:
Optional: since this is the uniform distribution, we can calculate true moments analytically.